PointerStar/Server/PointerStar.Server.csproj

```diff
@@ -6,6 +6,7 @@
 
   <ItemGroup>
     <PackageReference Include="Hashids.net" Version="1.6.1" />
+    <PackageReference Include="Majorsoft.Blazor.Components.Modal" Version="1.5.0" />
```

Comment:
Do we need this in the Server project? I would only expect it to be needed in the Client project